source/hic_hal/stm32/stm32h743ii/extended_features/source/udb_power_measurement.c

```diff
@@ -0,0 +1,89 @@
+#include "udb_power_measurement.h"
+#include "pac193x.h"
+#include <stdbool.h>
+#include <stdint.h>
+
+static pac193x_cfg_t s_udb_power_measurement_cfg;
+
+static const float k_udb_power_measurement_resister[] =
+{
+    [UDB_POWER_MEASUREMENT_TARGET_UDB] = 0.015f,
```

Comment:
Note: P2 boards are 0.015 ohm, the p1 boards are 0.03 ohm.  
I'm in favor of deprecating the P1 boards, vs. adding more logic for them.